pkg/storage/truncate.go

```diff
@@ -36,6 +36,7 @@ func (s *Store) TruncateAllTablesDANGEROUS(logger *zap.Logger) error {
 	trb_admin_notes,
 	trb_advice_letters,
 	trb_request_system_intakes,
+	trb_request_contract_numbers,
```

Comment:
Can you add this to `scripts/dev` like you did here for system intake contract numbers?

https://github.com/CMSgov/easi-app/blob/dc989e930736eba9955ad49be4e090cd992861cc/scripts/dev#L373